app/controllers/employees_controller.rb

```diff
@@ -33,6 +35,7 @@ def update
     @employee = Employee.find(params[:id])
     @employee.slack_username = params[:employee][:slack_username]
     @employee.validate_slack_username_in_org
+    @employee.add_slack_user_id_to_employee
```

Comment:
Yeah, I agree that it's getting a little bloated. I could make a new issue for this if you want? I'm not sure I have a ton of time to get to this particularly soon